en/docs/develop/streaming-apps/creating-a-Siddhi-Application.md

```diff
@@ -362,10 +364,14 @@ To create a Siddhi application via the design view of the Streaming Integrator T
 10. To align the Siddhi components that you have added to the grid,
     click **Edit** and then click **Auto-Align**. As a result, all the
     components are horizontally aligned as shown below.  
-    ![Aligned Siddhi components](../images/Creating-Siddhi-Applications/Siddhi_Application_Design_View.png)
+    
+    ![Aligned Siddhi components](../../assets/img/streaming/Creating-Siddhi-Applications/Siddhi_Application_Design_View.png)
+    
 11. Click **Source View**. The siddhi application is displayed as follows.  
-    ![Source view](../images/Creating-Siddhi-Applications/Siddhi_Application_Source_View.png)
+
+    ![Source view](../../assets/img/streaming/Creating-Siddhi-Applications/Siddhi_Application_Source_View.png)
+    
 12. Click **File** and then click **Save as**. The **Save to Workspace** dialog box appears. In the **File Name** 
     field, enter `SweetProductionAnalysis` and click **Save**.
 
-    ![Saving the Siddhi application](../images/Creating-Siddhi-Applications/Save_Siddhi_Application.png)
+    ![Saving the Siddhi application](../../assets/img/streaming/Creating-Siddhi-Applications/Save_Siddhi_Application.png)
```

Comment:
@RukshiW, the path needs to be as follows:

{{base_path}}/assets/img/streaming/Creating-Siddhi-Applications/Save_Siddhi_Application.png

In addition, there should not be any capital letters in the image files or document file names, as we run a script at build time to convert caps to lower case and when that conversion takes place the image links will break.

Please consider these comments for all the other image links as well.

Reply:
Corrected as described for the Develop section.